### PR TITLE
Change to builtin types as preferred since python 2.2

### PR DESCRIPTION
**Pull request by ericvaandering**
Please merge when Jenkins is OK.

### CODE REVIEW

File: src/python/WMCore/Configuration.py [hunk 1]
```diff
@@ -17,19 +17,19 @@
...
-    types.LongType,
-    types.NoneType,
-    types.IntType,
+    bool,
+    float,
+    bytes,
+    str,
+    int,
+    type(None),
+    int,
```
Comment (amaltaro) [inline src/python/WMCore/Configuration.py]: not a big deal, but 'int' twice here

Comment (ericvaandering) [inline src/python/WMCore/Configuration.py]: Yeah, because Long became int. It happened one other place too. Let me fix it.

**Comment by ericvaandering** [comment]
Really this whole commit points out problems that really should be solve with isintance, I think.

**Comment by ericvaandering** [comment]
test this please

**Comment by ericvaandering** [comment]
These unit tests breakages are legit. Do not merge.

**Pull request closed without merging**